### DISCUSSION TITLE
Ollama integration

### DISCUSSION THREAD

**Discussion by jariass2** [ChatGPTNextWeb/NextChat#3484] [Q&A] [answered]
Is there anyone who can help me to setup chatGPT with ollama server function

**Comment by jmatzen** [comment]
100% needs this feature ASAP

**Reply by LeaveNhA** [reply]
Any act on this? We can discuss and create a road-map on this feature.

**Comment by senscarlos** [comment]
Maybe it's just enough to put LiteLLM in the middle and change the URL.

**Reply by H0llyW00dzZ** [reply]
yeah I agree with this, it so easy hahahaha 100+ LLM

**Reply by LeaveNhA** [reply]
Yeah, actually I did it. 🙃

**Comment by fred-bf** [comment, marked as answer]
Check this out: https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/issues/4030
We will track the model support progress here, it take time to refactor the codes to make adjustments

**Reply by fred-bf** [reply]
@jariass2 jariass2 As for Ollama support, current blocker is the tauri requires cors response which Ollama not able to configure. If you are using webpages with a public url, you should be able to use ollama directly

**Comment by H0llyW00dzZ** [comment]
Ollama is not even stable yet, especially for their performance

**Comment by lloydzhou** [comment]
try on v2.13.0, see #5001

**Comment by benshgit** [comment]
100% needs this feature ASAP
